spylib/token.py

```diff
@@ -265,6 +276,76 @@ async def execute_gql(
 
         return jsondata['data']
 
+    async def create_http_webhook(
+        self,
+        topic: Union[WebhookTopic, str],
+        callback_url: str,
+        include_fields=None,
+        metafield_namespaces=None,
+        private_metafield_namespaces=None,
+    ) -> WebhookResponse:
+        """Uses graphql to create a webhook"""
+        query = '''
+        mutation webhookSubscriptionCreate($topic: WebhookSubscriptionTopic!,
+                                           $webhookSubscription: WebhookSubscriptionInput!) {
+            webhookSubscriptionCreate(topic: $topic, webhookSubscription: $webhookSubscription) {
+                webhookSubscription {
+                id
+                topic
+                format
+                endpoint {
+                    __typename
+                    ... on WebhookHttpEndpoint {
+                    callbackUrl
+                    }
+                  }
+                }
+                userErrors {
+                    field
+                    message
+                }
+              }
+            }
+        '''
+        variables = {
+            'topic': topic.value if isinstance(topic, WebhookTopic) else topic,
+            'webhookSubscription': {
+                'callbackUrl': callback_url,
+                'format': WEBHOOK_JSON_FORMAT,
```

Comment:
I think setting this directly in the query is probably fine

Reply:
Yeah it's constant/hardcoded in both cases.
And if it becomes a variable later for some reason it'll need to be refactored in both cases.